test/integration/api/forms/draft.js

```diff
@@ -406,6 +406,32 @@ describe('api: /projects/:id/forms (drafts)', () => {
               .set('Content-Type', 'application/xml')
               .expect(200)))));
 
+
+      it('should reject new draft with missing meta group', testService(async (service) => {
```

Comment:
What happens if you have a form that doesn't have a `meta` group and you want to create a new form version that includes `meta`? Are you able to use Frontend to create a draft to do so, or will that request fail?

Reply:
Oh, good point. It will probably be bad. I'm not sure how to even write a test for this.

One way around it is only check for the missing meta field on a new form. Then if they delete it in a later version, they'll see a warning about the structure changing, though they won't be blocked from it.

Reply:
I think that would be a nice improvement!